src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = days * 40;
+
+  if (days >= 7) {
+    return result - 50;
+  } else if (days >= 3) {
+    return result - 20;
```

Comment:
I think it would be useful to extract every number into separate `const` variable - I don't really know what 7 or 50 means, please provide variables like `const DAILY_RATE = 40` etc. - [checklist ](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)point 2